back-end/api/controllers/projects.controller.js

```diff
@@ -26,9 +26,37 @@ export default class ProjectsController {
      */
     static async apiDeleteProject(req, res, next) {
         try {
-            const projectName = req.query.project_name
+            const projectName = req.body.project_name
             console.log(projectName)
-            const projectResponse = await ProjectsDAO.deleteProject(projectName)
+            await ProjectsDAO.deleteProject(projectName)
+            res.json({ status: "success" })
+        } catch (e) {
+            res.status(500).json({ error: e.message })
+        }
+    }
+
+    static async apiUpdateProject(req, res, next) {
```

Comment:
Make sure to add comments about what every method/function is doing. How is this methd updating the project?